docs/intro/go.md

```diff
@@ -0,0 +1,95 @@
+---
+order: 2
+description: Install the Go programming language
+---
+
+# Install Go
+
+In this tutorial, you will install the Go programming language (Golang) on your local computer. Follow the instructions for your operating system:
+
+## macOS
+
+* Download the latest MacOS installer package from <https://golang.org/dl/>.
+* Open the downloaded package and follow the prompts through to completion.
+* By default, the package installs the Go distribution to `/usr/local/go`, however it is always best to define the path explicitly:
+
+    1. Open or create a `~/.profile` file with your favorite command-line text editor.
+    2. Add the following lines:
+
+        ```sh
+        export PATH=$PATH:$(go env GOPATH)/bin
+        ```
+
+    3. To make sure these changes are applied, run the following command:
+
+        ```sh
+        source ~/.profile
+        ```
+
+* Verify the installation of go by checking its version:
```

Comment:
```suggestion
* Verify that you have installed Go by checking running the following command to check the version:
```

Reply:
it's so tempting to use possessive "its" so I usually try to rephrase a sentence to avoid